protobuf/flow/entities/execution_result.proto

```diff
@@ -0,0 +1,29 @@
+syntax = "proto3";
+
+package flow.entities;
+
+option go_package = "github.com/onflow/flow/protobuf/go/flow/entities";
+option java_package = "org.onflow.protobuf.entities";
+
+message ExecutionResult {
+  bytes previous_result_id = 1;
+  bytes block_id = 2;
+  repeated Chunk chunks = 3;
+  repeated ServiceEvent service_events = 4;
+}
+
+message Chunk {
+  uint32 CollectionIndex = 1;
+  bytes start_state = 2; // state when starting executing this chunk
+  bytes event_collection = 3; // Events generated by executing results
+  bytes block_id = 4; // Block id of the execution result this chunk belongs to
+  uint64 total_computation_used = 5; // total amount of computation used by running all txs in this chunk
+  uint32 number_of_transactions = 6; // number of transactions inside the collection
```

Comment:
i noticed this went from uint64 in flow-go to uint32 here. Is the thought that this just won't need that much number space?

I feel it might still be better to match types, even if it's excessive

Reply:
Yeah, actually I noticed this isn't the only case where this is true either. There are a few other cases where the field doesn't match that in the Go model

Reply:
> Is the thought that this just won't need that much number space?

Yes, I guess we always said 1 000 000 is max number of tx per block.
I'm not sure if its worth expanding to uint64 here or simply reducing to uint32 in main flow repo

Reply:
Reverted to uint64 to match whats in 'flow-go'